source/accessible-media/index.html.erb.md

```diff
@@ -19,55 +19,50 @@ contributors:
+-   [**Transcripts**](transcript.html) are a text version of a video’s audio track,
+    containing narration, dialog and non-speech information such as
+    speaker-identification cues. Transcripts can be static or
+    interactive documents. Interactive transcripts can be synchronized to the media: words are highlighted automatically as they are spoken, or
+    users can select words to move to that spot in the
     video.
+-   [**Audio descriptions**](production-audio-description.html) are an extra track of narration that conveys
+    information about important visuals, such as body
+    language, changes in scenery and context, charts and
+    diagrams.
 
-## Quality considerations
+## What are good quality captions and audio descriptions?
 
-The quality of the captions and descriptions that are delivered to users
-is of high importance. In the same way that low-quality, incomplete
+Low-quality, incomplete
 audio is unacceptable to hearing viewers, captions that do not
-accurately reflect the audio, or contain spelling and grammatical
+accurately reflect the audio, contain spelling, or grammatical
 errors, are unacceptable to deaf and hard-of-hearing viewers. Similarly,
 audio descriptions that provide misleading or inaccurate information, or
 which provide unnecessary detail (e.g., by giving away clues or
-answers), are of no use to viewers who are blind or visually impaired.
+answers), are of less use to viewers who are blind or visually impaired.
 
 ## Why is this important?
 
 Besides providing crucial communication access for viewers who are deaf, hard of hearing, blind or visually impaired, captions and descriptions have benefits beyond their original target
 audiences:
 
--   Both captions and descriptions can be used to help students and adults with cognitive disabilities engage more directly with media, helping to focus attention or to provide comprehension support.
+-   Students and adults with cognitive disabilities can engage more directly with media when captions and audio descriptions are available as they help to focus attention or provide comprehension support.
 -   Captions are useful when teaching children and adults how to read,
-    or to help people learn a foreign language.
--   Captions can be used by all people: for example, in noisy
-    environments such as a bar or an airport lounge, or in any place
+    or to enable people learn a foreign language.
+-   Everyone can use captions in noisy
+    environments such as a bar, an airport lounge, or in any place
     where it would be inappropriate to turn up the volume.
--   Search engines can use captions as a way to locate information in a
-    video or audio presentation.
--   Audio descriptions can be used to reinforce complex visual concepts,
-    especially for children or adults with learning disabilities.
-
-In some cases, captions and descriptions may be the only means by which users can access any information at all from video or audio clips.
```

Comment:
Again, I’m not sure if that sentence adds anything to the page that is not clear from the previous content of the page.